upgrade/upgrade-scripts/common.sh

```diff
@@ -442,6 +442,15 @@ function fix_and_migrate_services() {
 		fi
 	fi
 
+	#
+	# Due to https://github.com/influxdata/telegraf/issues/14052, telegraf must be masked after
+	# packages are upgraded. The telegraf package removes /etc/systemd/system/telegraf.service thus
+	# reversing the `systemctl mask` operation performed before the packages are upgraded.
+	#
+	if [[ "$(systemctl is-enabled telegraf)" == enabled ]]; then
```

Comment:
Do we still need this block since you added the one in `execute`?

Reply:
@sebroy `fix_and_migrate_services` is also called when a container is created [here](https://github.com/delphix/appliance-build/blob/master/upgrade/upgrade-scripts/upgrade-container#L447). While a typical upgrade workflow will always involve running `execute` immediately after the container is created, I figured there is no harm in leaving this here (to reduce noise within the container for example) so that telegraf is also disabled within the container immediately. Rarely, but I (and Prakash) have used the scripts to spin up an upgrade container by hand.